Update defender-for-office-365-whats-new.md (#11803)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/defender-for-office-365-whats-new.md

## April 2023

- [Using machine learning to drive more effective simulations in Attack Simulation and Training](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/attack-simulation-training-using-machine-learning-to-drive-more/ba-p/3791023): Make use of intelligent predicted compromise rate (PCR) and Microsoft Defender for Office 365 payload recommendations for utilizing high-quality payloads in your simulation. 

> - [Training only campaigns available with an expanded library](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/training-only-campaign-is-now-available-with-an-expanded/ba-p/3795237): Customers can now directly assign training content to their organization without needing to tie training to a phishing simulation campaign. We have also expanded our training module library to more than 70 different modules.

Reviewer: ```suggestion
- [Training only campaigns available with an expanded library](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/training-only-campaign-is-now-available-with-an-expanded/ba-p/3795237): You can now directly assign training content to your organization without needing to tie training to a phishing simulation campaign. We have also expanded our training module library to more than 70 different modules.
```
